packages/grid/src/features/standard-table/components/StandardTableHeadRow.tsx

```diff
@@ -52,7 +52,11 @@ export const StandardTableHeadRow = React.memo(function StandardTableHeadRow<
 
   return (
     <TableHeadRow
-      top={headerRowOffsetTop ?? 0}
+      top={
+        headerRowOffsetTop && groupConfigs
```

Comment:
`groupConfigs` is the wrong way to check if table has groups.

I have added some code to master.
Use `useColumnGroupOrderContext` and check if it returns undefined or not.
If undefined, groups are not enabled. If it is a list (empty or filled with groups), groups are enabled.

Reply:
Fixed!